apps/docs/content/components/Avatar.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Avatar
+status: Experimental
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=543%3A19830&t=Be91VGjX2BANJnQc-11'
+source: https://github.com/primer/brand/tree/main/packages/react/src/Avatar/Avatar.tsx
+storybook: '/brand/storybook/?path=/story/components-Avatar--playground'
+description: Use Avatar to display a thumbnail representation of a person.
+---
+
+import {Label} from '@primer/react'
+
+```js
+import {Avatar, Stack} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<Avatar src="https://i.pravatar.cc/160?u=80" alt="A random avatar picture" />
```

Comment:
Shall we use Mona here instead? For branding and also the URL is likely to be more reliable.

```suggestion
<Avatar src="https://avatars.githubusercontent.com/u/92997159?v=4" alt="Mona profile" />
```